### DISCUSSION TITLE
Get Pixel Spacing in Datastatd ?

### DISCUSSION THREAD

**Discussion by anais2390** [Project-MONAI/MONAI#3413] [Q&A]
Hi everyone, 
I would like to know if there is a way to print the pixel spacing in between the diffrent Transform of the data during preprocessing using Datastatd in the Compose class ? 
Thanks

**Comment by Nic-Ma** [comment]
Hi @anais2390 ,

Thanks for your interest here.
I think you can leverage the `additional_info` arg of this transform to extract expected data from the meta data (like keys="image_meta_dict").

Thanks.

**Reply by anais2390** [reply]
Thanks @Nic-Ma for you reactivity. 
Can you give an exemple on how to use `additional_info ` ?

For exemple I want to get size and pixel spacing of label before and after `Spacing `Transform. For now I write the following command, but it gives me only size. How can I get the spacing ? 

```
 Compose([
        LoadImaged(keys=["image", "label"]),
        EnsureChannelFirstd(keys=["image", "label"]), 
        ScaleIntensityd(keys=["image"]),
        DataStatsd(prefix="before spacing", keys=["label"]),
        Spacingd(keys=["image", "label"], pixdim=spacing, mode=['bilinear', 'nearest'], padding_mode='border', align_corners=True),   
        DataStatsd(prefix="afterspacing", keys=["label"]),
)]
```
